lib/generateMarkdown.js

```diff
@@ -103,14 +103,26 @@ function getRecursiveTOC(orderedTypes, parentType, depth) {
     for (var i = 0; i < orderedTypes[parentType].children.length; i++) {
         var currentType = orderedTypes[parentType].children[i];
         if (orderedTypes[currentType].parents.length == 1) {
-            md += style.bulletItem(style.linkType(style.typeValue(currentType), currentType, enums.autoLinkOption.codeQuoteOnly), depth);
+            md += style.bulletItem(getTOCLink(currentType.replace(parentType + " ", ""), currentType), depth);
             md += getRecursiveTOC(orderedTypes, currentType, depth + 1);
         }
     }
     return md;
 }
 
 /**
+* @function getTOCLink
```

Comment:
Markdown-specific logic (like forming links) should be in style.js.  And, I already had bad form by having two separate functions hardcode the `reference` string, and now you're adding a third.  Can you make a constant within style.js for that, and then use it in the two existing functions as well as your new one?